core/client/fs/src/test/java/alluxio/client/file/BaseFileSystemTest.java

```diff
@@ -452,37 +490,114 @@ public void unmountException() throws Exception {
   }
 
   /**
-   * Ensures Warnings are logged or exceptions are thrown when an {@link AlluxioURI} with a
-   * scheme://host is passed.
+   * Ensures warnings are logged and exception is thrown when an {@link AlluxioURI} with an invalid
+   * authority is passed.
    */
   @Test
-  public void UriCheckTest() throws Exception {
+  public void uriCheckBadAuthority() throws Exception {
     Configuration.set(PropertyKey.MASTER_HOSTNAME, "localhost");
     Configuration.set(PropertyKey.MASTER_RPC_PORT, "19998");
+    TestAppender appender = new TestAppender();
```

Comment:
Can we do the appender setup/teardown in the `@Before` and `@After` methods? Then we don't have to duplicate across tests, and the teardown will still happen even if the test throws an exception

Reply:
Could I add the configuration settings into the before method too?

Reply:
I'd recommend against this since it would apply to every test, and different tests might want different configuration